app/models/post.rb

```diff
@@ -1,2 +1,17 @@
 class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
[OPTIONAL] According to your DB schema, this line should be:
```rb
belongs_to :AuthorId, class_name: 'User'
```
Check out this article to learn more about that: [Use a custom column name with a belongs_to association in rails](https://stevepolito.design/blog/use-a-custom-column-name-with-a-belongs-to-association-in-rails/)